.gitignore

```diff
@@ -16,3 +16,4 @@
 coverage.txt
 codegpt
 dist
+.idea/*
```

Comment:
Unfortunately, the code patch provided is not a code change, but rather a git diff command. Without the actual code changes, it is impossible to perform a code review. Please provide the actual code changes for review.